app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (
+    VaccineError,
+    NotWearingMaskError
+)
```

Comment:
You have used parentheses while importing multiple classes/modules which is a good practice according to the checklist.